test/functional/core/startup_spec.lua

```diff
@@ -1097,3 +1097,23 @@ describe('user session', function()
     eq(1, eval('g:lua_session'))
   end)
 end)
+
+describe('inccommand on ex mode', function()
+  it('should not preview', function()
+    clear()
+    local screen
+    screen = Screen.new(60, 10)
+    screen:attach()
+    local id = funcs.termopen({ nvim_prog, '--clean', '-u', 'NONE', '-E', 'test/README.md' })
```

Comment:
Oh wait, actually `VIMRUNTIME` is what is missing here
```suggestion
    local id = funcs.termopen({ nvim_prog, '-u', 'NONE', '-E', 'test/README.md' }, {
      env = { VIMRUNTIME = os.getenv('VIMRUNTIME') }
    })
```

Reply:
yup .. i also find it just now.

Reply:
also failed but not relate i think. could you please review again.